### ISSUE TITLE
Feature request: add image to notification

### ISSUE THREAD

**Issue by adispi** [advplyr/audiobookshelf-app#715]
Can you please add images to the new podcast notification?

**Comment by advplyr** [comment]
I don't understand what this mean. What new podcast notification?

**Comment by adispi** [comment]
sorry, i will explain:
in the setting, under notification, when a new podcast episode is downloaded, a notification is sent out to your configured devices. there are varibles for texts line name and so on, and i was wandering if it is possible to add a photo of the podcast to make the notiofication better, like in the ARR apps.
an example of an ARR notification:
![Screenshot 2023-05-27 190157](https://github.com/advplyr/audiobookshelf-app/assets/1352350/7da98c57-7b15-4fc7-be08-7e0174d0b1de)

**Comment by advplyr** [comment]
Do you have AppRise setup and you are using the notifications now in Abs? I could include the URL to the image if that is what you are looking for.

**Comment by adispi** [comment]
yes i do, and yes i am, and yes that is exactly what i meant.

**Comment by advplyr** [comment]
This would be the image hosted on your server which is protected by an API token. For this we would have to open up an unprotected URL for images. I'm guessing Sonarr is using the image URL from whatever provider they got it from?

**Comment by adispi** [comment]
yes, sonarr is taking the image from thetvdb radarr from themoviedb and so on.
i do not know how, but i think you should be able to provide the image from the podcast rss feed, like you do for the images in the server?
